controllers/alerts/alerts.go

```diff
@@ -215,3 +217,18 @@ func createOperatorHealthStatusRule() monitoringv1.Rule {
 		Expr:   intstr.FromString(`label_replace(vector(2) and on() ((kubevirt_hco_system_health_status>1) or (count(ALERTS{kubernetes_operator_part_of="kubevirt", alertstate="firing", operator_health_impact="critical"})>0)) or (vector(1) and on() ((kubevirt_hco_system_health_status==1) or (count(ALERTS{kubernetes_operator_part_of="kubevirt", alertstate="firing", operator_health_impact="warning"})>0))) or vector(0),"name","kubevirt-hyperconverged","","")`),
 	}
 }
+
+func createSingleStackIPv6AlertRule() monitoringv1.Rule {
+	return monitoringv1.Rule{
+		Alert: singleStackIPv6Alert,
+		Expr:  intstr.FromString("kubevirt_hco_hyperconverged_single_stack_ipv6 == 1"),
```

Comment:
@dharmit  I think the metric name is incorrect here. Should be kubevirt_hco_single_stack_ipv6

Reply:
Sure. I simply took the cue from another alert defined a few lines earlier in the same file:
```go
Expr:  intstr.FromString("kubevirt_hco_hyperconverged_cr_exists == 0"),
```

Reply:
I think the problem was I had used `kubevirt_hco_single_stack_ipv6` in `metrics.go`, but `kubevirt_hco_hyperconverged_single_stack_ipv6` in `alerts.go`. I have modified as you suggested.